lib/stitches/auto_fill.py

```diff
@@ -57,7 +57,7 @@ def is_outline(self):
         return self.key in self.OUTLINE_KEYS
 
     def is_segment(self):
-        return self.key == self.SEGMENT_KEY
+        return self.key.startswith(self.SEGMENT_KEY)
```

Comment:
Do we need to do similar for `is_outline()`?

Reply:
Yes, luckily startswith works with tuples as well.